services/gateway/namespaces.js

```diff
@@ -33,9 +33,9 @@ const filterApis = (requiredApis, availableApis) => {
 };
 
 module.exports = filterApis(config.api.ws, {
-	'/rpc': () => registerApi('http-version1', { ...defaultConfig }),
-	'/rpc-v1': () => registerApi('http-version1', { ...defaultConfig }),
-	'/rpc-test': () => registerApi('http-test', { ...defaultConfig }),
+	'/rpc': () => registerApi(['http-version1', 'http-version1-compat'], { ...defaultConfig }),
+	'/rpc-v1': () => registerApi(['http-version1', 'http-version1-compat'], { ...defaultConfig }),
+	'/rpc-test': () => registerApi(['http-test'], { ...defaultConfig }),
```

Comment:
I think this array does not have to be obligatory there, can you make it support also a plain string when a single name is there?

Reply:
Done!